_docs/flavour/templates.md

```diff
@@ -2,14 +2,14 @@
 title: HTML templates
 ---
 
-Flavour provides convenient way to render and update DOM via HTML templates.
-All you need is to define class with data and behaviour and bind it to HTML template.
-This section describes following topics:
+Flavour provides a convenient way to render and update DOM via HTML templates.
+All you need is to define a class with data and behaviour, then bind it to an HTML template.
+This section describes the following topics:
 
 * how to create HTML templates;
 * how to use standard components;
 * how to create custom components;
-* how to use expression language.
+* how to use Flavour's expression language.
```

Comment:
Is reference 'Flavour' really needed here?

Reply:
Something is needed here, in my opinion, one of the following:
* an expression language
* the expression language
* its expression language